src/components/Help/Help.js

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
 import './Help.css';
+import xIcon from '../../assets/x-icon.png'
```

Comment:
We should actually move this into the public folder and call on it from there. That is where our images will live for the PWA integration